### ISSUE TITLE
(WCAG 2.1) Lack of role and State Indication for Expandable/Collapsible Control

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#5527]
OBSERVED RESULTS:

- The expandable/collapsible control labeled "What are the eligibility requirements?" does not provide a clear indication of its role and current expanded or collapsed state.
- ![Screenshot (1)](https://github.com/civiform/civiform/assets/86739416/484169c3-e0e7-441f-95fd-c4d14190b669)

EXPECTED RESULTS:

- The expandable/collapsible control should have a role button/link and clearly convey its current state (expanded or collapsed) to users, including those who rely on assistive technologies.
- Use appropriate ARIA attributes (e.g., aria-expanded) to dynamically convey the state of the control to screen readers.

STEPS TO REPRODUCE:

- Enable screen reader.
- Navigate to [URL](https://demo.civiform.dev/)
- Navigate to and activate the ‘Apply’ for the Comprehensive sample program.
- Navigate to and activate “What is your address?”.
- Navigate to expandable/collapsible control "What are the eligibility requirements? using keyboard TAB or SHIFT TAB key.
- Observe the control does not provide any role and state information.

ENVIRONMENT DETAILS:

- Device(s) (brand, OS version, test key or release key): Windows 10 Enterprise (64-bit),MacOS Ventura Version 13.4
- Screen reader name and version: JAWS 2022.2211.7, NVDA 2022.3, VoiceOver
- Browser name and version: Chrome Version 113.0.5672.93\

**Comment by melanie-exygy** [comment]
Just a note that the state is currently present (`aria-expanded`).  See screenshot below and note the aria-expanded attribute in the developer tools.
![image](https://github.com/civiform/civiform/assets/134556723/2d0f4eaf-efad-4f54-a8a3-21ee6005c060)

**Comment by melanie-exygy** [comment]
Completed in this [PR](https://github.com/civiform/civiform/pull/5463).
